daml-assistant/BUILD.bazel

```diff
@@ -30,10 +30,6 @@ da_haskell_library(
     name = "daml-lib",
     srcs = glob(
         ["src/**/*.hs"],
-        exclude = [
```

Comment:
What’s up with this? Why are we including these two files now? This looks like we are now compiling them twice, once as part of `daml-lib` and then once for the tests and ones for the executable.

Reply:
These two files don't exist. I expect they were previously relocated as:
```
daml/daml-assistant/exe/DA/Daml/Assistant.hs
daml/daml-assistant/test/DA/Daml/Assistant/Tests.hs
```
I removed the unnecessary exclude directive just to avoid future confusion (I was confused when I saw it!)